sonarpedia-secrets.json

```diff
@@ -0,0 +1,11 @@
+{
+  "rules-metadata-path": "./sonar-text-plugin/src/main/resources/org/sonar/l10n/secrets/rules/secrets",
```

Comment:
I think it would be easier to do the same as what was done on sonarjs with CSS: Their is only a small folder with the sonarpedia file, and they keep one at the root. See https://github.com/SonarSource/SonarJS/tree/master/css-sonarpedia